qtrio/_qt.py

```diff
@@ -25,20 +26,34 @@ class Signal:
     _attribute_name: typing.ClassVar[str] = ""
 
     def __init__(self, *args: object, **kwargs: object) -> None:
+        from qts import QtCore
```

Comment:
do we need the import again?

Reply:
The import at the top of the file is only `if typing.TYPE_CHECKING:`.  It might be nice for us to look into 'lazy imports' or such but for now this is what we can do to have type hints referencing qt at the top level but not actually import qt until it is actually needed at runtime.  It's not beautiful.

Reply:
https://docs.python.org/3.9/library/typing.html#typing.TYPE_CHECKING